plugin/src/main/java/io/jenkins/plugins/analysis/core/model/DetailsTableModel.java

```diff
@@ -159,7 +166,8 @@ protected TableRow(final AgeBuilder ageBuilder,
                 final Issue issue,
                 final JenkinsFacade jenkinsFacade) {
             this.jenkinsFacade = jenkinsFacade;
-            description = formatDetails(issue, descriptionProvider.getDescription(issue));
+            descriptionContent = getDetailsContent(issue, descriptionProvider.getDescription(issue));
```

Comment:
Wouldn't it make more sense to use only the message? For Spotbugs, Checkstyle etc. there is a large amount of text that might match for everything?